models/get-models.js

```diff
@@ -41,3 +41,15 @@ exports.selectArticles = () => {
       });
     });
 };
+
+exports.selectComments = (article_id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE article_id = $1`, [article_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
+      rows.sort((a, b) => a.created_at - b.created_at);
```

Comment:
Instead of using javascript to sort the data use an SQL query. You can do it in the same query as above!